src/controllers/guests/posts.rs

```diff
@@ -154,7 +111,6 @@ pub async fn get_category_posts(
     let pages_count: Vec<_> = (1..=posts_per_page).collect();
     let mut count_of_number_of_pages = pages_count.len();
```

Comment:
`count_of_number_of_pages` is exactly the same as `posts_per_page`, isn't it ? Besides, it seems `pages_count` wasn't referenced any else where, can we remove it ?
```rust

```

Reply:
fixed it